.github/semantic.yaml

```diff
@@ -0,0 +1,44 @@
+---
+# Always validate the PR title, and ignore the commits
```

Comment:
can you add a source for this file that you referred to write this configuration?

Reply:
It is listed in the commit message. There is no full example (or I did not find it), only snippets on the site.